examples/tensorflow/multi-gpu-tensorflow.ipynb

```diff
@@ -0,0 +1,207 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "16caad50-4916-4d98-999e-e5bfb8a7cedb",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "def train_multigpu(n_epochs, classes, base_lr, batchsize, scale_batch=False, scale_lr=False):\n",
+    "\n",
+    "    strategy = tf.distribute.MirroredStrategy()\n",
+    "    print(\"Number of devices: %d\" % strategy.num_replicas_in_sync)\n",
+    "\n",
+    "    with strategy.scope():\n",
+    "        model = tf.keras.applications.ResNet50(include_top=True, weights=None, classes=classes)\n",
+    "\n",
+    "        optimizer = keras.optimizers.Adam(lr=base_lr)\n",
+    "        model.compile(\n",
+    "            loss=\"sparse_categorical_crossentropy\", optimizer=optimizer, metrics=[\"accuracy\"]\n",
+    "        )\n",
+    "\n",
+    "    # Data\n",
+    "    train_ds = (\n",
+    "        tf.keras.preprocessing.image_dataset_from_directory(\n",
+    "            \"dataset/birds/train\", image_size=(224, 224), batch_size=batchsize\n",
+    "        )\n",
+    "        .prefetch(2)\n",
+    "        .cache()\n",
+    "        .shuffle(1000)\n",
+    "    )\n",
+    "\n",
+    "    # printing sample birds images\n",
+    "    for birds, labels in train_ds.take(1):\n",
+    "        plt.figure(figsize=(18, 18))\n",
+    "        for i in range(9):\n",
+    "            plt.subplot(3, 3, i + 1)\n",
+    "            plt.imshow(birds[i].numpy().astype(\"uint8\"))\n",
+    "            plt.axis(\"off\")\n",
+    "    plt.show()\n",
+    "\n",
+    "    valid_ds = tf.keras.preprocessing.image_dataset_from_directory(\n",
+    "        \"dataset/birds/valid\", image_size=(224, 224), batch_size=batchsize\n",
+    "    ).prefetch(2)\n",
+    "\n",
+    "    start = time.time()\n",
+    "\n",
+    "    model.fit(\n",
+    "        train_ds,\n",
+    "        epochs=n_epochs,\n",
+    "        validation_data=valid_ds,\n",
+    "    )\n",
+    "\n",
+    "    end = time.time() - start\n",
+    "    print(\"model training time\", end)\n",
+    "\n",
+    "    tf.keras.models.save_model(model, \"model/keras_multi/\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f27c9a2d-b590-44da-bc39-f2a137143556",
+   "metadata": {},
+   "source": [
+    "In the code below we are setting up necessary parameters. We are only running a few epochs, to save time, but once you've got this working you'll have all the information you need to build and run bigger TensorFlow models on Saturn Cloud. If required increase batch size to take full advantage of multi-GPU processing power so that GPUs can be kept busy (but without running out of RAM). "
```

Comment:
```suggestion
    "In the code below, we set up necessary parameters. We run only a few epochs to save time. But once you've got this model working, you'll have all the information you need to build and run bigger TensorFlow models on Saturn Cloud. If required, increase the batch size to take full advantage of multi-GPU processing power so that GPUs can be kept busy (but without running out of RAM). "
```